h2o-py/h2o/frame.py

```diff
@@ -1299,7 +1299,7 @@ def as_data_frame(self, use_pandas=True, header=True):
         if can_use_pandas() and use_pandas:
             import pandas
             return pandas.read_csv(StringIO(self.get_frame_data()), low_memory=False, skip_blank_lines=False)
```

Comment:
Pandas provide their own built-in types to handle non-ascii chars. This branch is ok.